exercises/space-age/test/space_age_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:test/test.dart';
+import 'package:space_age/space_age.dart';
+
+final SpaceAge myAge = new SpaceAge();
+
+void main() {
+  group("SpaceAge: Planet - ", ageOnPlanetTests);
+}
+
+ageOnPlanetTests() {
+  test("Earth", () {
+    myAge.inSeconds = 1000000000;
```

Comment:
The canonical-data.json calls for the values for planet and seconds to be passed to a method called age.

Additionally, the value for planet would be of type String. The enums can still be used in the SpaceAge class, but to remain consistent with the canonical-data.json, the test gives the planet name as a String.

Reply:
Thinking about it now, I think it's fine to leave the enums in the test file.